autoreject/viz.py

```diff
@@ -1219,8 +1219,8 @@ def _plot_histogram(params):
         color = colors[types[idx]]
         rej = None
         if epochs.reject is not None and types[idx] in epochs.reject.keys():
```

Comment:
`types[idx] in epochs.reject.keys()`

is evil ! just do:

`types[idx] in epochs.reject`

Reply:
I suppose then a PR needs to be done in MNE: https://github.com/mne-tools/mne-python/blob/master/mne/viz/epochs.py#L1954 ;)

also why is it evil? It only works when you review, I have to give the reason when I review code ;-)

Reply:
in python 2 when you do:

epochs.reject.keys()

you get a list so search the presence in a list is **slow** while it's O(1) in a dict. I've seen in the past code that was super slow just because of this. Please change and send a PR to mne-python to fix. thanks

Reply:
sent PR upstream: https://github.com/mne-tools/mne-python/pull/6012/

and also fixed it here. Thanks for the explanation!